.eslintrc.json

```diff
@@ -0,0 +1,156 @@
+//     "indent": [1, 2],
+//     "one-var": 0,
+//     "one-var-declaration-per-line": 0,
+//     "new-cap": 0,
+//     "consistent-return": 0,
+//     "no-param-reassign": 0,
+//     "comma-dangle": 0,
+//     "curly": ["error", "multi-line"],
+//     "no-shadow": ["error", { "allow": ["req", "res", "err", "user", "role", "document"] }],
+//     /*"valid-jsdoc": ["error", {
+//       "requireReturn": true,
+//       "requireReturnType": true,
+//       "requireParamDescription": false,
+//       "requireReturnDescription": true
+//     }],
+//     "require-jsdoc": ["error", {
+//         "require": {
+//             "FunctionDeclaration": true,
+//             "MethodDefinition": true,
+//             "ClassDeclaration": true
+//         }
+//     }],*/
+//     "quotes": 0,
+//     "no-console": 1,
+//     "no-debugger": 1,
+//     "no-var": 1,
+//     "semi": [
+//       1, "always"
+//     ],
+//     "no-trailing-spaces": 0,
+//     "eol-last": 0,
+//     "no-unused-vars": 0,
+//     "no-underscore-dangle": 0,
+//     "no-alert": 0,
+//     "no-lone-blocks": 0,
+//     "jsx-quotes": 1,
+//     "react/display-name": [
+//       1, {
+//         "ignoreTranspilerName": false
+//       }
+//     ],
+//     "react/forbid-prop-types": [
+//       1, {
+//         "forbid": ["any"]
+//       }
+//     ],
+//     "jsx-a11y/no-static-element-interactions": [
+//       0,
+//       {
+//         "handlers": [
+//           "onClick",
+//           "onMouseDown",
+//           "onMouseUp",
+//           "onKeyPress",
+//           "onKeyDown",
+//           "onKeyUp"
+//         ]
+//       }
+//     ],
+//     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+//     "react/prefer-stateless-function": [0, { "ignorePureComponents": true }],
+//     "react/jsx-boolean-value": 1,
+//     "react/jsx-closing-bracket-location": 0,
+//     "react/jsx-curly-spacing": 1,
+//     "react/jsx-indent-props": 0,
+//     "react/jsx-key": 1,
+//     "react/jsx-max-props-per-line": 0,
+//     "react/jsx-no-bind": 0,
+//     "react/jsx-no-danger": 0,
+//     "react/jsx-no-duplicate-props": 1,
+//     "react/jsx-no-literals": 0,
+//     "react/jsx-no-undef": 1,
+//     "react/jsx-pascal-case": 1,
+//     "react/jsx-sort-prop-types": 0,
+//     "react/jsx-sort-props": 0,
+//     "react/jsx-uses-react": 1,
+//     "react/jsx-uses-vars": 1,
+//     "react/no-danger": 0,
+//     "react/no-did-mount-set-state": 1,
+//     "react/no-did-update-set-state": 1,
+//     "react/no-direct-mutation-state": 1,
+//     "react/no-multi-comp": 1,
+//     "react/no-set-state": 0,
+//     "react/no-unknown-property": 1,
+//     "react/prefer-es6-class": 1,
+//     "react/prop-types": 1,
+//     "react/react-in-jsx-scope": 1,
+//     "react/self-closing-comp": 1,
+//     "react/sort-comp": 1,
+//     "react/wrap-multilines": 1,
+//     "react/no-find-dom-node": 0,
+//     "react/no-string-refs": 0,
+//     "import/prefer-default-export": 0
+//   }
+// }
```

Comment:
The piece of code above seem to be commented out. If that is the case, then it has no reason to be there.